rpc/src/main/scala/org/bitcoins/rpc/client/v16/BitcoindV16RpcClient.scala

```diff
@@ -0,0 +1,88 @@
+package org.bitcoins.rpc.client.v16
+
+import akka.stream.ActorMaterializer
+import org.bitcoins.core.crypto.ECPrivateKey
+import org.bitcoins.core.protocol.transaction.Transaction
+import org.bitcoins.rpc.client.common.{
+  BitcoindRpcClient,
+  BitcoindVersion,
+  RpcOpts
+}
+import org.bitcoins.rpc.config.BitcoindInstance
+import org.bitcoins.rpc.jsonmodels.SignRawTransactionResult
+import org.bitcoins.rpc.serializers.BitcoindJsonSerializers._
+import play.api.libs.json.{JsArray, JsString, Json}
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * This class is compatible with version 0.16 of Bitcoin Core.
+  */
+class BitcoindV16RpcClient(override val instance: BitcoindInstance)(
+    implicit
+    m: ActorMaterializer)
+    extends BitcoindRpcClient(instance)
+    with V16AccountRpc
+    with V16SendRpc {
+
+  override val version: BitcoindVersion = BitcoindVersion.V16
+  require(version == instance.getVersion, "bitcoind version must be 0.16")
```

Comment:
`instance.getVersion` does `bitcoind --version` e.g. an IO operation. Is that problematic?

Reply:
I don't think so